web-services/query/src/test/java/datawave/webservice/query/runner/ExtendedQueryExecutorBeanTest.java

```diff
@@ -394,7 +394,7 @@ public void testAdminCancel_LookupAccumuloQuery() throws Exception {
         expect(this.queryLogicFactory.getQueryLogic("ql1", principal)).andReturn((QueryLogic) this.queryLogic1);
         expect(this.queryLogic1.getConnectionPriority()).andReturn(Priority.NORMAL);
         expect(this.queryLogic1.getCollectQueryMetrics()).andReturn(false);
-        expect(this.queryLogic1.isLongRunningQuery()).andReturn(false);
+        // expect(this.queryLogic1.isLongRunningQuery()).andReturn(false);
```

Comment:
what happened here?  same question for the subsequent commented out expect statements.

Reply:
It looks like this was something introduced in the `release/version3.42` branch. I rebased this PR to `release/version3.41` and these unit test modifications were no longer needed. Also, after backing out all changes from `release/version3.42` these unit test modifications were still required, so I'm going to look more closely at the `release/version3.42` branch to see what is happening there separately from this PR. 

Bottom line, I suspect these unit tests are failing in `release/version3.42` _in its current state_ without these changes.

Reply:
Confirmed `release/version3.42` was indeed bad due to a bad merge. Addressed this in https://github.com/NationalSecurityAgency/datawave/commit/cc0b9ea18895792118e90df31f6da38af6f538fe